File: vSongApp/src/main/java/com/jackson_siro/visongbook/models/Callback/CallbackPostsSlider.java
package com.jackson_siro.visongbook.models.Callback;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

import com.jackson_siro.visongbook.models.PostsSlider;

public class CallbackPostsSlider implements Serializable {
    public List<PostsSlider> data = new ArrayList<PostsSlider>();
}

File: vSongApp/src/main/java/com/jackson_siro/visongbook/models/callback/CallbackPostsSlider.kt
package com.jackson_siro.visongbook.models.callback

import java.util.ArrayList
import com.jackson_siro.visongbook.models.PostsSlider
import java.io.Serializable

internal class CallbackPostsSlider : Serializable {
    var data: List<PostsSlider> = ArrayList()
}
